tests/serverpod_test_server/test_integration/list_include_test.dart

```diff
@@ -0,0 +1,697 @@
+import 'package:serverpod/serverpod.dart' as pod;
+
+import 'package:serverpod_test_server/src/generated/protocol.dart';
+import 'package:serverpod_test_server/test_util/test_serverpod.dart';
+
+import 'package:test/test.dart';
+
+void main() async {
+  var testServer = IntegrationTestServer();
+  var session = await testServer.session();
+
+  tearDown(() async {
+    await Person.db.deleteWhere(session, where: (t) => pod.Constant.bool(true));
+    await Organization.db
+        .deleteWhere(session, where: (t) => pod.Constant.bool(true));
+    await City.db.deleteWhere(session, where: (t) => pod.Constant.bool(true));
+  });
+
+  test(
+      'Given a list relation of a city that can hold citizens where the database only contains the city when including the citizens list then an empty list is included in the result.',
```

Comment:
Not sure I agree here, while I get that it is indeed long I don't think the above description gives you enough information to understand the test. I prefer having the description in the description rather as a comment too.

Reply:
Another suggestion in that case, leave out the mention of the City and Citizens since they are only implementation details.


`Given object with list relation with nothing attached when fetching object including list relation then returned object has empty list as list relation`